src/main/webapp/addSchoolInfo.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>School Form</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <p>Add A Course Below</p>
+    <form action="/add-info" method="POST">
+        <p>School Name</p>
+        <textarea name="school-name">SchoolName</textarea>
+        <br><br>
+        <p>Course Name</p>
+        <textarea name="course-name">CoureName</textarea>
+        <br><br>
+        <p>Term</p>
+        <textarea name="term">Term</textarea>
+        <br><br>
+        <p>Unit</p>
+        <textarea name="units">Units</textarea>
```

Comment:
Sidenote: I've been using checkboxes instead of textareas for my units section and I think that makes it easier in the long run (since you won't run into people putting in invalid numbers), maybe consider changing to that?

Reply:
I changed it to a number input box so they can only input a number from 1 - 24